EIPS/eip-asset_bound_non_fungible_tokens.md

```diff
@@ -0,0 +1,524 @@
+---
```

Comment:
```suggestion
---
eip: 6956
```
Issuing EIP number 6956 using PR number.
Please also update the filename

Reply:
Thank you - refactored Files, Interface-Names etc. Also took the opportunity to unify Event- and function naming.

Sorry for the many force-pushes below, eth-bot put up quite a fight (which I appreciate as it refuses to let my improperly formatted code base pass)

I suppose conversations in PR here are resolved by the person who started them, @abcoathup ?